src/runtime/particle-spec.ts

```diff
@@ -347,4 +366,30 @@ export class ParticleSpec {
   toManifestString(): string {
     return this.toString();
   }
+
+  private _parseTrustClaims(claims: SerializedParticleTrustClaimSpec[]): Map<string, string> {
```

Comment:
This isn't parsing; parsing is converting tokens to an AST. check? validate?

Reply:
done